src/server/pps/cmds/cmds.go

```diff
@@ -235,6 +237,8 @@ $ pachctl list-job -p foo bar/YYY
 		}),
 	}
 	rawFlag(listDatum)
+	listDatum.Flags().BoolVar(&paginate, "paginate", false, "Get a subset of the results")
+	listDatum.Flags().Int64Var(&page, "page", 0, "Specify the page of results to send")
```

Comment:
Ah, I just saw this. [I actually made the change and it wans't any additional casts really](https://github.com/pachyderm/pachyderm/commit/a8e5e199a50dd18aac808c60bc163762036bb926)

But if you want int64, I can do that. I will have to add some error checking though (right now if you specify a negative page, I believe we'll panic). Using `uint64` seems like an easier way to deal to validate the input consistently.